recipes/forestdb/all/test_package/conanfile.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from conans import ConanFile, CMake, tools, RunEnvironment
+from conan.tools.build import cross_building
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
```

Comment:
```suggestion
    generators = "cmake", "cmake_find_package_multi"
```
It will generate xxx-config.cmake files.